### DISCUSSION TITLE
Apple m1 error

### DISCUSSION THREAD

**Discussion by brylos1** [vuejs/vue-cli#6577] [Q&A]
I installed the latest version of node for apple m1. When I try to install vue, I get such a dependency error as in the picture. anyone know how to fix it?![image](https://user-images.githubusercontent.com/60017069/124681963-0135df80-deca-11eb-8993-6a50308ac15f.png)

**Comment by ekhodx** [comment]
Mine works well. Give more info please. Which command do you use? Do you have rosetta or smth installed? Also it may be important in which terminal you execute this commands

update: oops, it seems i am late for answering this question. sorry for bumping then

**Comment by Viggo95** [comment]
Everything works fine for me
